### PR TITLE
Add backup job.

### PR DESCRIPTION
**Pull request by HumairAK**
Resolve: https://github.com/operate-first/apps/issues/2077

### CODE REVIEW

**Review by harshad16 — COMMENTED** [review]
lgtm
great work 
just one question

**Review by harshad16 — COMMENTED** [review]
:smiley:

File: vault/overlays/moc/smaug/backup-job/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: trigger-backup-job
+spec:
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  schedule: "0 */2 * * *"
```
[the remark this answers is not in the document]
Comment (HumairAK) [reply vault/overlays/moc/smaug/backup-job/cronjob.yaml:8]: we don't, I forgot to update this -- I'll make it twice a day.

[the remark this answers is not in the document]
Comment (HumairAK) [reply vault/overlays/moc/smaug/backup-job/cronjob.yaml]: updated

[the remark this answers is not in the document]
Comment (harshad16) [reply vault/overlays/moc/smaug/backup-job/cronjob.yaml]: we can start with that,
i just feel the secret update in vault will not happen this frequent, so may once we observe the backup, maybe we can reduce it to one daily or weekly.

[the remark this answers is not in the document]
Comment (HumairAK) [reply vault/overlays/moc/smaug/backup-job/cronjob.yaml]: yeah we still have yet to update all the secrets, so there will be a large amount of updates happening in the coming week, we can reduce it after we're done converting all secrets and confirm the backups are working as intended.

**Pull request merged**